runtime/common/src/paras_registrar.rs

```diff
@@ -271,6 +283,30 @@ decl_module! {
 			ensure_root(origin)?;
 			Self::remove_lock(para);
 		}
+
+		/// Register a Para Id on the relay chain.
+		///
+		/// This function will queue the new Para Id to be a parathread.
+		/// Using the Slots pallet, a parathread can then be upgraded to get a
+		/// parachain slot.
+		///
+		/// This function must be called by a signed origin.
+		///
+		/// The origin must pay a deposit for the registration information,
+		/// including the genesis information and validation code. ParaId
+		/// must be greater than or equal to 1000.
```

Comment:
```suggestion
		/// including the genesis information and validation code. The ParaId
		/// will be set to the next available id.
```

Do we maybe want to dispatch an event with the assigned paraid?